util/chplenv/chpl_atomics.py

```diff
@@ -38,6 +38,8 @@ def get(flag='target'):
                     atomics_val = 'intrinsics'
                 elif version >= CompVersion('4.1') and not platform_val.endswith('32'):
                     atomics_val = 'intrinsics'
+            elif compiler_val == 'aarch64-gnu':
+                atomics_val = 'cstdlib'
```

Comment:
Why cstdlib here, shouldn't this also be based on whatever check we end up adding for detecting cstdlib support?